ion/src/device/shared/usb/dfu_relocated.cpp

```diff
@@ -59,7 +59,7 @@ void DFU(bool exitWithKeyboard, bool unlocked, int level) {
 
   /* 4- Disable all interrupts
    * The interrupt service routines live in the Flash and could be overwritten
-   * by garbage during a firmware upgrade operation, so we disable them. */
+   * by garbage during a firmware upgrade opration, so we disable them. */
```

Comment:
Why do you edit it ?
```suggestion
   * by garbage during a firmware upgrade operation, so we disable them. */
```